src/data/balance/starters.ts

```diff
@@ -0,0 +1,1224 @@
+  [Species.SLITHER_WING]: Abilities.SCRAPPY,
+  [Species.SANDY_SHOCKS]: Abilities.EARTH_EATER,
+  [Species.IRON_TREADS]: Abilities.STEELY_SPIRIT,
+  [Species.IRON_BUNDLE]: Abilities.SNOW_WARNING,
+  [Species.IRON_HANDS]: Abilities.IRON_FIST,
+  [Species.IRON_JUGULIS]: Abilities.LIGHTNING_ROD,
+  [Species.IRON_MOTH]: Abilities.LEVITATE,
+  [Species.IRON_THORNS]: Abilities.SAND_STREAM,
+  [Species.FRIGIBAX]: Abilities.SNOW_WARNING,
+  [Species.GIMMIGHOUL]: Abilities.HONEY_GATHER,
+  [Species.WO_CHIEN]: Abilities.VESSEL_OF_RUIN,
+  [Species.CHIEN_PAO]: Abilities.INTIMIDATE,
+  [Species.TING_LU]: Abilities.STAMINA,
+  [Species.CHI_YU]: Abilities.BERSERK,
+  [Species.ROARING_MOON]: Abilities.TOUGH_CLAWS,
+  [Species.IRON_VALIANT]: Abilities.ADAPTABILITY,
+  [Species.KORAIDON]: Abilities.OPPORTUNIST,
+  [Species.MIRAIDON]: Abilities.OPPORTUNIST,
+  [Species.WALKING_WAKE]: Abilities.BEAST_BOOST,
+  [Species.IRON_LEAVES]: Abilities.SHARPNESS,
+  [Species.POLTCHAGEIST]: Abilities.TRIAGE,
+  [Species.OKIDOGI]: Abilities.FUR_COAT,
+  [Species.MUNKIDORI]: Abilities.NEUROFORCE,
+  [Species.FEZANDIPITI]: Abilities.LEVITATE,
+  [Species.OGERPON]: Abilities.OPPORTUNIST,
+  [Species.GOUGING_FIRE]: Abilities.BEAST_BOOST,
+  [Species.RAGING_BOLT]: Abilities.BEAST_BOOST,
+  [Species.IRON_BOULDER]: Abilities.SHARPNESS,
+  [Species.IRON_CROWN]: Abilities.SHARPNESS,
+  [Species.TERAPAGOS]: Abilities.SOUL_HEART,
+  [Species.PECHARUNT]: Abilities.TOXIC_CHAIN,
+  [Species.ALOLA_RATTATA]: Abilities.ADAPTABILITY,
+  [Species.ALOLA_SANDSHREW]: Abilities.ICE_SCALES,
+  [Species.ALOLA_VULPIX]: Abilities.SHEER_FORCE,
+  [Species.ALOLA_DIGLETT]: Abilities.STURDY,
+  [Species.ALOLA_MEOWTH]: Abilities.DARK_AURA,
+  [Species.ALOLA_GEODUDE]: Abilities.DRY_SKIN,
+  [Species.ALOLA_GRIMER]: Abilities.TOXIC_DEBRIS,
+  [Species.ETERNAL_FLOETTE]: Abilities.MAGIC_GUARD,
+  [Species.GALAR_MEOWTH]: Abilities.STEELWORKER,
+  [Species.GALAR_PONYTA]: Abilities.MOXIE,
+  [Species.GALAR_SLOWPOKE]: Abilities.UNAWARE,
+  [Species.GALAR_FARFETCHD]: Abilities.INTREPID_SWORD,
+  [Species.GALAR_ARTICUNO]: Abilities.SERENE_GRACE,
+  [Species.GALAR_ZAPDOS]: Abilities.TOUGH_CLAWS,
+  [Species.GALAR_MOLTRES]: Abilities.DARK_AURA,
+  [Species.GALAR_CORSOLA]: Abilities.SHADOW_SHIELD,
+  [Species.GALAR_ZIGZAGOON]: Abilities.POISON_HEAL,
+  [Species.GALAR_DARUMAKA]: Abilities.FLASH_FIRE,
+  [Species.GALAR_YAMASK]: Abilities.TABLETS_OF_RUIN,
+  [Species.GALAR_STUNFISK]: Abilities.ARENA_TRAP,
+  [Species.HISUI_GROWLITHE]: Abilities.RECKLESS,
+  [Species.HISUI_VOLTORB]: Abilities.TRANSISTOR,
+  [Species.HISUI_QWILFISH]: Abilities.MERCILESS,
+  [Species.HISUI_SNEASEL]: Abilities.SCRAPPY,
+  [Species.HISUI_ZORUA]: Abilities.ADAPTABILITY,
+  [Species.PALDEA_TAUROS]: Abilities.ADAPTABILITY,
+  [Species.PALDEA_WOOPER]: Abilities.THICK_FAT,
+  [Species.BLOODMOON_URSALUNA]: Abilities.BERSERK
+};
+
+const starterCandyCosts: { passive: integer; costReduction: [integer, integer]; egg: integer; }[] = [
```

Comment:
Please no more `integer`:
```suggestion
const starterCandyCosts: { passive: number; costReduction: [number, number]; egg: number; }[] = [
```